init.sql

```diff
@@ -159,3 +159,202 @@ create table asset_creators
+);
+
+create table candy_machine_data
+(
+    id                         bigserial        PRIMARY KEY,
+    uuid                       varchar(6),
+    price                      int,
+    symbol                     varchar(5)       not null,
+    seller_fee_basis_points    int              not null,
+    max_supply                 int              not null,
+    is_mutable                 bool             not null,
+    retain_authority           bool,
+    go_live_date               int,
+    items_available            int              not null,
+    candy_machine_id           bytea references candy_machine (id),
+);
+create unique index candy_machine_data_candy_machine_id on candy_machine_data (candy_machine_id);
+
+create table candy_machine_creators
+(
+    id                    bigserial                                PRIMARY KEY,
+    candy_machine_id      bytea references candy_machine (id)      not null,
+    creator               bytea                                    not null,
+    share                 int                                      not null default 0,
+    verified              bool                                     not null default false
+);
+create unique index candy_machine_creators_candy_machine_id on candy_machine_creators (candy_machine_id);
+create index candy_machine_creator on candy_machine_creators (candy_machine_id, creator);
+create index candy_machine_verified_creator on candy_machine_creators (candy_machine_id, verified);
+
+create table candy_guard
+(   
+    id                   bytea                                   PRIMARY KEY,
+    bump                 int                                     not null,
+    authority            bytea                                   not null,
+)
+
+create table candy_guard_group
+(
+    id                   bigserial                              PRIMARY KEY,
+    label                varchar(50)                            not null,
+    candy_guard_id       bytea references candy_guard (id)    not null,
+)
+
+create table candy_machine_whitelist_mint_settings
+(
+    id                           bigserial                                                PRIMARY KEY,
+    candy_machine_id             bytea references candy_machine (id),
+    candy_guard_group            int references candy_guard_group (id),
+    mode                         whitelist_mint_mode                                      not null,
+    mint                         bytea                                                    not null,
+    presale                      bool                                                     not null,
+    discount_price               int
+) 
+create unique index candy_machine_whitelist_mint_settings_candy_machine_id on candy_machine_whitelist_mint_settings (candy_machine_id);
+create unique index candy_machine_whitelist_mint_settings_candy_guard_group on candy_machine_whitelist_mint_settings (candy_guard_group);
+
+-- TODO should version be an enum on cm table
```

Comment:
Is this TODO going to be resolved as part of this PR?

Reply:
i think that version can be removed entirely tbh. based on the latest pr init sql